Week3/homework/step2-5.js

```diff
@@ -4,8 +4,13 @@ function multiplyAll(arr) {
   // eslint-disable-next-line
   let product = 1;
 
-  // Replace this comment and the next line with your code
-  console.log(arr, product);
+  for (let x = 0; x < arr.length; x++) {
+    let innerProduct = 1;
+    for (let y = 0; y < arr[x].length; y++) {
+      innerProduct = innerProduct * arr[x][y];
```

Comment:
There is no real need to use an `innerProduct` intermediate variable here. You can update the `product` variable directly, and use the short-hand notation:

```js
product *= arr[x][y];
```

Note also that most programmers use the letter i-k rather x and y for loop indices (but don't use `l`: it looks too much like `1`  (one)).

Reply:
Done